.github/workflows/azure-dev.yml

```
@@ -42,12 +42,14 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container:
-      image: mcr.microsoft.com/azure-dev-cli-apps:latest
 
     steps:
```

```suggestion
    steps:
      - name: Checkout
```

Remove redundant newline